conftest.py

```diff
@@ -150,7 +150,7 @@ def make_valid_request(send_post):
     def _make_valid_request(method: SensorMethod, params: dict | None = None) -> dict:
         payload = make_valid_payload(method=method, params=params)
         sensor_response = send_post(**payload)
-        return sensor_response.get("result", {})
+        return sensor_response.get("result", "error")
```

Comment:
Good idea to make explicit cases when response from sensor doesn't contain `result`. However, in the next homework we'll be altering this line, so you don't really need to change it in order for new tests in this PR to work properly.

Instead of checking for `error` in your tests you could simply check for an empty dict, since that's what this function is going to return when sensor responds with an error.